src/ng2-smart-table/lib/data-set/column.ts

```diff
@@ -10,7 +10,7 @@ export class Column {
   public isFilterable: boolean = false;
   public sortDirection: string = '';
   public defaultSortDirection: string = '';
-  public editor: {type: string, config: any} = {type: '', config: {}};
+  public editor: {type: string, config: any, component: any} = {type: '', config: {}, component: null};
```

Comment:
- please, in objects add space before first property and after last like `{ type: string, config: any, component: any }`